packages/inventory/src/components/detail/AppInfo.js

```diff
@@ -9,21 +9,28 @@ import { Skeleton, SkeletonSize } from '@redhat-cloud-services/frontend-componen
  * This component detail is accessed from redux if no component found `missing component` is displayed.
  * @param {*} props `componentsMapper` if you want to pass different components list.
  */
-const AppInfo = ({ componentsMapper }) => {
+const AppInfo = ({ componentMapper, appList }) => {
     const store = useStore();
     const loaded = useSelector(({ entityDetails: { loaded } }) => loaded);
+    const entity = useSelector(({ entityDetails: { entity } }) => entity);
     const activeApp = useSelector(({ entityDetails: { activeApps, activeApp, loaded } }) => {
         if (loaded) {
-            return activeApps?.find?.(item => item?.name === activeApp?.appName) || activeApps?.[0];
+            return (appList || activeApps)?.find?.(item => item?.name === activeApp?.appName) || activeApps?.[0];
         }
     });
-    const Cmp = componentsMapper?.[activeApp?.name] || activeApp?.component;
+    const Cmp = componentMapper || activeApp?.component;
```

Comment:
How does this work if the `componentMapper` is an `object`? I think it will trigger the `cannot render objects as children` error. Or is the object prop type referring to react component?

Reply:
Oh yeah, I should remove the object from the props. Good point!